superset-frontend/packages/superset-ui-core/src/query/getClientErrorObject.ts

```diff
@@ -51,6 +53,72 @@ type ErrorType =
 
 type ErrorTextSource = 'dashboard' | 'chart' | 'query' | 'dataset' | 'database';
 
+const ERROR_CODE_LOOKUP = {
+  400: 'Bad request',
+  401: 'Unauthorized',
+  402: 'Payment required',
+  403: 'Forbidden',
+  404: 'Not found',
+  405: 'Method not allowed',
+  406: 'Not acceptable',
+  407: 'Proxy authentication required',
+  408: 'Request timeout',
+  409: 'Conflict',
+  410: 'Gone',
+  411: 'Length required',
+  412: 'Precondition failed',
+  413: 'Payload too large',
+  414: 'URI too long',
+  415: 'Unsupported media type',
+  416: 'Range not satisfiable',
+  417: 'Expectation failed',
+  418: "I'm a teapot",
```

Comment:
🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖 🫖

Reply:
TIL https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418

Reply:
Haha, yeah i had to research this one